test/relations.test.js

```diff
@@ -2527,42 +2527,6 @@ describe('relations', function() {
         done();
       });
     });
-
-    it('should allow to create belongsTo model in beforeCreate hook', function(done) {
```

Comment:
Uff, this is a weird test. Is it related to https://github.com/strongloop/loopback-datasource-juggler/pull/934/files#r63366803? Could you please check whether it is possible to reproduce the same behaviour using operation hooks?